lib/paper_trail/attribute_serializers/object_changes_attribute.rb

```diff
@@ -8,6 +8,12 @@ module AttributeSerializers
     class ObjectChangesAttribute
       def initialize(item_class)
         @item_class = item_class
+
+        # ActiveRecord since 7.0 has a built-in encryption mechanism
+        @encrypted_attributes =
+          if PaperTrail.active_record_gte_7_0?
+            @item_class.encrypted_attributes&.map(&:to_s)
```

Comment:
Surprisingly, it [seems](https://github.com/search?q=repo%3Arails%2Frails%20encrypted_attributes&type=code) that rails does not initialize `encrypted_attributes` to an empty array, and so the safe-navigation operator is warranted.